README.md

```diff
@@ -55,13 +55,25 @@ Before using ECP with your application/client, you should complete the policy co
 
 ### Manual Certificate Configuration
 
-ECP relies on the `certificate_config.json` file to read all the metadata information for locating the certificate.
-It is stored as a JSON file at the following location on the user's device:
+ECP relies on a certificate configuration JSON file to read all the metadata information for locating the certificate.
+By default, it is named `certificate_config.json` and stored at the following location on the user's device:
 
-* **Linux and MacOS**: ~/.config/gcloud/certificate_config.json
-* **Windows**: %APPDATA%\gcloud\certificate_config.json 
+* **Linux and MacOS**: `~/.config/gcloud/certificate_config.json`
+* **Windows**: `%APPDATA%\gcloud\certificate_config.json`
 
-Below are example configurations:
+You can put the JSON file in the location of your choice and set the path to it using:
+
+```
+$ gcloud config set context_aware/certificate_config_file_path "<json file path>"
```

Comment:
I think we still need to implement this

Reply:
(For future reference, this was implemented within the Google Cloud SDK itself.)